test/v2_integration.py

```diff
@@ -1173,6 +1173,25 @@ def test_auth_deactivation_v2():
         raise Exception("unexpected authorization status")
 
 
+def check_ocsp_successful(cert_file, issuer_file, url):
```

Comment:
```suggestion
def simple_check_ocsp_successful(cert_file, issuer_file, url):
```

minor nit: I think having "simple" (or something else) in the name helps make it easier to understand the function has some trade-offs/limitations. The function docstring does a good job of explaining those but if you're only looking at a call site it might be tempting to crib it for use in other contexts without thinking to check the limitations. WDYT?

Reply:
_(Probably shouldn't use a Github suggestion for this since it will break the call site if applied in isolation from the web UI, sorry)_

Reply:
Agreed that the name could convey that this is only inspecting a few bytes and not doing full validation.

Reply:
Good suggestion. I went with check_ocsp_basic_oid but happy to go for `simple_check...` if you think that's better.

Reply:
`check_ocsp_basic_oid` works for me. Thanks!